Added details on how to configure Entra OIDC (#870)
wandb/docs: docs/guides/hosting/iam/sso.md

_wandb/local_ uses Open ID Connect (OIDC) for authentication. Based on your use case, select one of the tabs to learn how to set up AWS Cognito or Okta authenticate W&B Server with Open ID Connect.

<Tabs
  defaultValue="aws"
  values={[
    {label: 'AWS', value: 'aws'},
    {label: 'Okta', value: 'okta'},

>      {label: 'Entra', value: 'entra'},

Reviewer: ```suggestion
    {label: 'Okta', value: 'okta'},
    {label: 'Entra', value: 'entra'},
```

7. Provide the OIDC Issuer, Client ID, and Auth method to wandb/local on https://deploy.wandb.ai/system-admin or set them as environment variables.

  </TabItem>

<TabItem value="entra">
1. Login to the [Azure Portal](https://portal.azure.com/#home).

2. Select "Microsoft Entra ID" service.
![](/images/hosting/entra_select_entra_service.png)

> 3. On the left side, select "App registrations".

Reviewer: ```suggestion
3. On the left side, select "App registrations."
```

![](/images/hosting/entra_app_registrations.png)

> 4. On the top, click "New registration".

Reviewer: ```suggestion
4. On the top, click "New registration."
```

![](/images/hosting/entra_new_app_registration.png)

>     On the screen named "Register an application", fill out the values as follows:

Reviewer: ```suggestion
    On the screen named "Register an application," fill out the values as follows:
```

![](/images/hosting/entra_register_an_application.png)

>     - Specify a name, e.g. "Weights and Biases application".

Reviewer: ```suggestion
    - Specify a name, for example "Weights and Biases application".
```

>     - Leave the default account type: "Accounts in this organizational directory only (Default Directory only - Single tenant)". Modify if you need to.

Reviewer: ```suggestion
    - Leave the default account type: "Accounts in this organizational directory only (Default Directory only - Single tenant)." Modify if you need to.
```

Reviewer: @mkaesz-wandb "Leave the default" and "modify if you need to" are contradictory. Which is it?

    - Configure Redirect URI as type "Web" with value: `https://YOUR_W_AND_B_URL/oidc/callback`

>     - Click "Register".

Reviewer: ```suggestion
    - Click "Register."
```

> - Make a note of the "Application (client) ID" and "Directory (tenant) ID". 

Reviewer: ```suggestion
- Make a note of the "Application (client) ID" and "Directory (tenant) ID."
```

![](/images/hosting/entra_app_overview_make_note.png)

> 5. On the left side, click "Authentication"

Reviewer: ```suggestion
5. On the left side, click **Authentication**.
```

![](/images/hosting/entra_select_authentication.png)

> - Under "Front-channel logout URL" specify: `https://YOUR_W_AND_B_URL/logout`

Reviewer: ```suggestion
- Under **Front-channel logout URL**, specify: `https://YOUR_W_AND_B_URL/logout`
```
